backend/Car-Detailing-Backend/Data/Models/list_of_services.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Car_Detailing_Backend.Data.Models
+{
+    public class list_of_services
+    {
+        public int service_id { get; set; }
+        public string name { get; set; }
```

Comment:
In class naming, let us use the CamelCase naming convention.
Also, what do you think about using `Records` for this type of class? I believe it can provide a more clear way of writing data classes and simplify the code